mobify/platform-scaffold
# [Integration Manager] Miscellaneous command conversions

This PR converts the logout and search suggestion operations from built-in requests to Integration Manager commands. Only an implementation for Merlin's is included at the moment. 

 **JIRA**: N/A (preparatory for merging the IM)

## Changes
- Implement logout in Merlin's as a command
- Implement search suggestions in Merlin's as a command. 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure that log out still functions
- Ensure that search suggestions still appear.
